dmap.js

```diff
@@ -1,9 +1,9 @@
 module.exports = lib = {}
 
 const multiformats = require('multiformats')
-const prefLenIndex = 29
-const hashLenIndex = 30
-const lockIndex = 31
+const prefLenIndex = 2
+const hashLenIndex = 3
+lib.FLAG_LOCK = 1 << 7
```

Comment:
Don’t get why it’s this instead of just `1`
It’s right everywhere else here so not sure how this even works 
Can investigate properly later

Reply:
Ok gotcha